Timeline.Core/Timeline.cs

```diff
@@ -3605,14 +3606,24 @@ private void LoadSingle(string path)
             {
                 document.Load(path);
                 ReadInterpolableTree(document.FirstChild, dic, _selectedOCI);
-
+#if KOIKATSU || SUNSHINE
+                OCIChar character = _selectedOCI as OCIChar;
+                StudioResolveInfo resolveInfo = UniversalAutoResolver.LoadedStudioResolutionInfo.FirstOrDefault(x => x.Slot == document.FirstChild.ReadInt("animationNo") && x.GUID == document.FirstChild.Attributes?["GUID"]?.InnerText && x.Group == document.FirstChild.ReadInt("animationGroup") && x.Category == document.FirstChild.ReadInt("animationCategory"));
```

Comment:
`document.FirstChild`, `document.FirstChild.Attributes?["GUID"]?.InnerText` and `document.FirstChild.Read*` are expensive to run in a loop, read them into variables and reuse.